src/clr-angular/utils/_reboot.clarity.scss

```diff
@@ -97,9 +97,9 @@
   // might still respond to pointer events.
   //
   // Credit: https://github.com/suitcss/base
-  [tabindex='-1']:focus {
-    outline: none !important;
-  }
+  //[tabindex='-1']:focus {
+  //  outline: none !important;
+  //}
```

Comment:
I couldn't override this reboot's global setting by trying to change it in menu mixin. I think we would like to allow outline in tabindex=-1 cases globally anyways because those are the ones we'd programmatically manipulate to focus. Let me know if anyone thinks differently.

Reply:
Thanks for the comment. As this reboot was taken from bootstrap originally, I thought all the commented out parts were left intentional with comments above them to show that it was done intentionally (you can see examples of doing that throughout the file). Do you think there's value in doing that? I can either add a comment above it (I don't yet have it) or remove the commented out lines as you suggest.